docs/personalization/tracking_with_ibexa-tracker.md

```diff
@@ -18,12 +18,15 @@ Otherwise a new one is created.
 
 The Personalization JavaScript is a Google-like tracking API (ga.js) that you can paste 
 into your pages. 
-It activates the tracking by inserting <https://cdn.perso.ibexa.co/ibexa-tracker.js> 
-into the page.
+It activates the tracking by inserting <https://cdn.perso.ibexa.co/ibexa-tracker.js> or
+<https://cdn.perso.ibexa.co/ibexa-tracker.min.js> into the page.
 
 To use this mechanism on your pages, copy the code snippet below, and replace 
-`<YOUR_MANDATOR_ID>` with the customer ID, `<PAGE_ID>` with the page/item identifier 
-and `<USER_ID>` with the one generated by your user identifier. 
+`<YOUR_MANDATOR_ID>` with the customer ID, `<PAGE_ID>` with the page/item identifier.
+`<USER_ID>` - empty string for cookie based anonymous user or replaced with the value generated
+by your user identifier system for logged-in user. In case user activity should not be tracked,
+'not_defined' value should be used to still have an event stored without link to the user.
```

Comment:
```suggestion

- `<YOUR_MANDATOR_ID>` with the customer ID
- `<CONTENT_TYPE_ID>` with the Content Type ID
- `<CONTENT_ID>` with the content item ID
- `<USER_ID>` with an empty string for cookie based anonymous user or replaced with the value generated by your user identifier system for logged-in user. In case user activity should not be tracked, 'not_defined' value should be used to still have an event stored without a link to the user.
```

1) I think a list makes it more readable
2) The PAGE_ID placeholder doesn't exist anymore, but there's CONTENT_ID and CONTENT_TYPE_ID - can you please check if my suggestion makes sense?
3) Please also add a `:` after replace (I cannot add it through GitHub suggsetions)

Reply:
Chaged according to suggestion.
About keep only minified version, I think it will be good for debug purpose to have the original file. It could help partners easy undersdtand how tracking working.